package.json

```diff
@@ -185,6 +185,11 @@
         "command": "azure-iot-edge.showGallery",
         "title": "Show Sample Gallery",
         "category": "Azure IoT Edge"
+      },
+      {
```

Comment:
If triggered from command palette, what will happen?

Reply:
I will disable command palette

Reply:
Updated

Reply:
Seems no need to regist command here, refer to "azure-iot-edge.internal.addModule"

Reply:
Oh, I will remove it